lib/app/utils/validation.rb

```diff
@@ -54,6 +54,15 @@ def self.get_resource(json, version)
       snf: 'https://bluebutton.cms.gov/assets/ig/StructureDefinition-bluebutton-snf-claim'
     }.freeze
 
+    US_CORE_R4_URIS = {
```

Comment:
I think this file is very problematic because it tightly couples core Inferno behavior to specific sequences. I don't know that it needs to be fixed in this PR, but I think a better solution would be for `SequenceBase` to have a default implementation that just looks at the profile the resource declares, and for sequences to override that method as needed (probably by creating and including a module).

Reply:
I'll make a ticket for this.